README.md

```diff
@@ -184,8 +184,6 @@ The enabled features for a crate now are resolved at build time! That means you
   build generation time since we can simply pass this set to `cargo metadata`. Feature selection during build time is
   out of scope for now.~~
 * No support for building and running tests, see [nixpkgs, issue 59177](https://github.com/NixOS/nixpkgs/issues/59177).
-* [Renamed crates](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#renaming-dependencies-in-cargotoml)
```

Comment:
Can you please mention from what version on this is supported -- in line with the other issues? Thank you!

Reply:
What version should I mention here? Do you plan to release 0.5.1 next or go to 0.6.0 (since renames introduce an API change in the form of extra arguments to buildRustCrate).

Reply:
I have now put *Before 0.6.x* here.